quasar-utils/javac-plugin/src/main/java/net/corda/plugins/javac/quasar/SuspendableDatabase.java

```diff
@@ -0,0 +1,77 @@
+package net.corda.plugins.javac.quasar;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Set;
+
+public class SuspendableDatabase {
```

Comment:
This class has been derived from Quasar's `co.paralleluniverse.fibers.instrument.SimpleSuspendableClassifier`, but has not preserved Ron Pressler's copyright header. This looks like a licence violation to me.

Reply:
that's correct, I will add it